misc/roundtrip

```diff
@@ -43,9 +43,9 @@ fi
 limit_entries()
 {
     if [ -n "${MAX_ENTRIES}" ]; then
-	head -${MAX_ENTRIES}
+	    head -${MAX_ENTRIES}
```

Comment:
Cold you remove this change from the commit?
I believe FreeBSD is not so whtespace sensitive.